data/actions/actions.xml

```diff
@@ -38,6 +38,8 @@
 	<action fromid="2096" toid="2097" script="other/decayto.lua" />
 	<action fromid="2162" toid="2163" script="other/decayto.lua" />
 	<action itemid="2578" script="other/decayto.lua" />
+	<action itemid="7184" script="other/decayto.lua" />
+	<action itemid="6572" script="other/decayto.lua" />
```

Comment:
I don't quite understand this.

We already have action id 6572 in music.lua - this will throw a duplicate action id error aswell

Sorry if I'm missing something here.